docs/Getting_Involved/Testing.rst

```diff
@@ -9,8 +9,15 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
+If you have not already, you can start by following
+the `coala Tutorial <Users/Tutorials/Tutorial.html>`_.
+
+Before you run tests, make sure you
+`install coala
+<Users/Install.html/>`_.
```

Comment:
btw you don't need to :O tests work without coala installed (at least they should)